frontend/lib/swr-cache/index.ts

```diff
@@ -72,18 +75,41 @@ export const withCache = <
    const logger: Logger = defaultLog;
 
    const requestRevalidation = async (variables: z.infer<VariablesSchema>) => {
-      fetch(`${APP_ORIGIN}/${endpoint}`, {
-         method: 'POST',
-         body: JSON.stringify(variables),
-      }).catch((error) => {
-         logger.error(
-            `${endpoint}.error: failed to trigger revalidation\n${printError(
-               error
-            )}`
-         );
+      const start = performance.now();
+      await new Promise<void>((resolve, reject) => {
+         // Don't delay the lambda beyond 50ms for revalidation
+         const timeout = setTimeout(() => {
+            const elapsed = performance.now() - start;
+            logger.warning.event(`${endpoint}.revalidation.timeout`);
+            logger.warning.timing(`${endpoint}.revalidation.timeout`, elapsed);
+            resolve();
+         }, 50);
+         const postData = JSON.stringify(variables);
+         const req = request({
```

Comment:
I agree that we should keep things simple, but the docs indicate that setting the `Content-Length` header does have a change in behavior:

![image](https://user-images.githubusercontent.com/26855/232913994-740209c9-1f9f-4b77-b089-3bf205d40981.png)

I don't know if it'll impact performance, but it seems reasonable to test it in preview.